src/components/chat/chat-thread.scss

```
@@ -0,0 +1,64 @@
+@import "../vars";
+
+.chat-list {
+    height:100%;
+    overflow-y: scroll;
+    overflow-x: hidden;
```

Two-space indenting throughout file
```suggestion
.chat-list {
  height: 100%;
  overflow-y: scroll;
  overflow-x: hidden;
```